keyboards/40percentclub/4x4/keymaps/via/4x4.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This is a VIA json and as such needs to be removed, and pr'ed to `the-via/keyboards` instead.